### PR TITLE
Static Content and T & C Checkbox

### PR DESCRIPTION
**Pull request by Tinusw**
Ch, Ch, Ch, Ch, Changes...

**CreatedStatic Pages for**

1. Know what information is being placed in the public domain
static_pages/public.html.haml

2. Understand how my information will be used
static_pages/info_use.html.haml

3. Find out why I should add an event
static_pages/why.html.haml

4. Terms & Conditions
static_pages/terms_and_conditions.html.haml
Deploy actions

**Also added link_to for all static pages in navbar**

**Additions to Event Create form**

- Checkbox for Terms and Conditions with model validation.
- Also has link_to Terms and Conditions that opens in new tab.

checbox_validation

## Checklist
Make sure these boxes are checked before submitting your contribution

- [X] This PR has only the commits related to my branch
- [X] I've done self review and my code looks great!
- [X] I've tested my changes and written tests when applicable

### CODE REVIEW

File: app/views/events/new.html.haml [hunk 1]
```diff
@@ -38,7 +38,13 @@
           .form-group
             = f.label :owner_phone
             = f.text_field :owner_phone, class: 'form-control'
-
+          .form-group
+            = f.label :terms_and_conditions do
+              .form-inline
+                I hereby accept the 
+                = link_to 'Terms and Conditions ', static_pages_terms_and_conditions_path, target: '_blank'
```
Comment (Janther) [inline app/views/events/new.html.haml]: space after `Conditions`

File: app/views/events/new.html.haml [hunk 2]
```diff
@@ -38,7 +38,13 @@
...
+                = f.check_box :terms_and_conditions, class: 'checkbox'
+              
           %button.btn.btn-primary{data: { target: "#locationModal", toggle: "modal" }, type: "button" }
```
Comment (Janther) [inline app/views/events/new.html.haml]: space missing after `{`

File: app/views/shared/_topnav.html.haml [hunk 3]
```diff
@@ -24,3 +24,20 @@
               = link_to 'View Events', events_path
             %li
               = link_to 'New Event', new_event_path
+        %li.dropdown
+          %a.dropdown-toggle{ href: '#',
+                              data: { toggle: 'dropdown' },
+                              role: 'button',
+                              aria: { haspopup: 'true', expanded: 'false' } }
+            %i= fa_icon 'info'
```
Comment (Janther) [inline app/views/shared/_topnav.html.haml]: Pretty sure `fa_icon` returns an `<i>` tag

File: app/views/events/new.html.haml (hunk 1 above)
Comment (Tinusw) [inline app/views/events/new.html.haml]: 👍

File: app/views/events/new.html.haml (hunk 2 above)
Comment (Tinusw) [inline app/views/events/new.html.haml]: 👍

File: app/views/shared/_topnav.html.haml (hunk 3 above)
Comment (Tinusw) [inline app/views/shared/_topnav.html.haml]: 👍

**Pull request merged**